### DISCUSSION TITLE
feature request: Comment

### DISCUSSION THREAD

**Discussion by Xaporia** [thegeeklab/hugo-geekdoc#433] [Ideas]
Is there any plan for possibility of comment or "ask and answer" part on the posts of blog ?

**Comment by xoxys** [comment]
Hi, thanks. We don't plan to integrate any kind of commenting system, sorry.

You might want to look into this topic https://gohugo.io/content-management/comments/. As Hugo supports to overload files from themes, integration it on your own might be possible without that much work. 

Disqus for example just need to be enabled in the config, and you can add the internal template by overwriting this template https://github.com/thegeeklab/hugo-geekdoc/blob/main/layouts/posts/single.html
